src/lib/y2storage/autoinst_profile/partition_section.rb

```diff
@@ -431,6 +431,18 @@ def lvm_group_name(device)
       def fixed_size?(device)
         device.is?(:disk_device, :software_raid)
       end
+
+      # Determines whether given partition is primary or not
+      #
+      # Always false when the partition belongs to a GPT partition table.
+      #
+      # @param partition [Y2Storgae::Partition] the partition to check
+      # @return [Boolean] true when is a primary partition; false otherwise
+      def primary_partition?(partition)
+        return false if partition.partition_table.type.is?(:gpt)
```

Comment:
It would be better to use something more generic like `PartitionTable#:extended_possible?`

Reply:
Thank you so much for the explanation, @ancorgs 

I am going to do a new PR fixing this :)